### PR TITLE
Set env configure for UserCount

### PR DESCRIPTION
**Pull request by vttranlina**

### CODE REVIEW

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Comment by chibenwa** [comment]
I would be glad to have a sample.env at the root of the project where I can discover easily, amongst other, such configuration options.

File: sample.env [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+export JMAP_PROTOCOL=https
+export WEBADMIN_PORT=80
+export WS_PROTOCOL=wss
+export DURATION=15
+export IMAP_PORT=993
+export IMAP_PROTOCOL=imaps
+export USER_COUNT=1000
+export SMTP_PORT=25
+export SMTP_SSL_PORT=465
+export WEBADMIN_PROTOCOL=http
```
Comment (chibenwa) [inline sample.env]:
```suggestion
#export TARGET_HOSTNAME=localhost
#export JMAP_TARGET_HOSTNAME=localhost
#export IMAP_SERVER_HOSTNAME=localhost
#export SMTP_TARGET_HOSTNAME=localhost
#export WEBADMIN_SERVER_HOSTNAMe=localhost
#export JMAP_PORT=443
#export JMAP_PROTOCOL=https
#export WEBADMIN_PORT=80
#export WS_PROTOCOL=wss
#export DURATION=15
#export IMAP_PORT=993
#export IMAP_PROTOCOL=imaps
#export USER_COUNT=1000
#export SMTP_PORT=25
#export SMTP_SSL_PORT=465
#export WEBADMIN_PROTOCOL=http
```

Let's comment those by default, an admin can then uncomment as needed

File: sample.env [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export TARGET_HOSTNAME=localhost
+export JMAP_TARGET_HOSTNAME=localhost
+export IMAP_SERVER_HOSTNAME=localhost
+export SMTP_TARGET_HOSTNAME=localhost
+export WEBADMIN_SERVER_HOSTNAMe=localhost
+export JMAP_PORT=443
+export JMAP_PROTOCOL=https
+export WEBADMIN_PORT=80
+export WS_PROTOCOL=wss
+export DURATION=15
```
Comment (chibenwa) [inline sample.env]: Can we have at least a comment giving the unit?

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by Arsnael** [comment]
Squash the fixup please :)

**Pull request merged**